nile-backend/src/test/java/com/tp/Nile/NileApplicationTests.java

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
I don't think we should comment this part out unless it is preventing us from being able to develop.